src/NuGetUtility/Output/Json/JsonOutputFormatter.cs

```diff
@@ -27,15 +27,9 @@ public async Task Write(Stream stream, IList<LicenseValidationResult> results)
         {
             if (_printErrorsOnly)
             {
-                IEnumerable<LicenseValidationResult> resultsWithErrors = results.Where(r => r.ValidationErrors.Any());
-                if (resultsWithErrors.Any())
-                {
-                    await JsonSerializer.SerializeAsync(stream, resultsWithErrors, _options);
-                    return;
-                }
+                results = results.Where(r => r.ValidationErrors.Any()).ToList();
```

Comment:
Please make sure the integration tests are updated and cover the newly added functionality...

Reply:
Sure, I need time to understand what the heck going on there. This is a bit new approach for me :)

Reply:
Ok, now I think I understand how it works. I've updated the verified results according to my changes. There are no new scenarios based on my changes. One more thing, I find it a bit strange that a package can be "ignored" and invalid at the same time. I think the test generator should be adjusted a bit but it's not a big deal.

Reply:
About the test generator. Logically this is correct - however this has no meaning to the result printer.